### ISSUE TITLE
Composer.json minimum stability

### ISSUE THREAD

**Issue by Orkin** [api-platform/schema-generator#188]
Hello, when you would like to update the package to the last tag (2.2.1) I got an error because of the minimum-stability: dev

My project has a constraint to minimum-stability: stable and when I try to update with Symfony 5 I can't because of this new requirement in the composer.json.

So it is possible to remove this like for the version 2.1 ?

I can make a PR for that if necessary

Thanks

**Comment by dunglas** [comment]
Can you copy the error you get? The `minimum-stability` setting of this package is used by composer only when installing directly the dependencies of the package (e.g. when developing it and in our CI), it shouldn't have any impact on your project.

**Comment by Orkin** [comment]
@dunglas I have something really strange, I tried with an empty composer.json to install this package with the version 2.1 everything is fine and if I tried with the last version I have something like this (I never seen this error before)

With `composer require --dev api-platform/schema-generator 2.1` everything is fine
With `composer require --dev api-platform/schema-generator` I got this error :
```
Problem 1
    - don't install api-platform/schema-generator v2.2.0|remove my-company/my-project dev-develop
    - don't install api-platform/schema-generator v2.2.1|remove my-company/my-project dev-develop
    - Installation request for my-company/my-project dev-develop -> satisfiable by my-company/my-project[dev-develop].
    - Installation request for api-platform/schema-generator ^2.2 -> satisfiable by api-platform/schema-generator[v2.2.0, v2.2.1].
```

_I just replaced the real package name of `my-company/my-project`_
The only difference I found in dependencies is the support for symfony 5 and the add of this minimum-stability but I'm agree with you, it shouldn't have any impact. This is really weird.

And this is my composer.json (I just removed unusefull informations)

```json
{
    "name": "my-company/my-project",
    "description": "My Project",
    "type": "project",
    "homepage": "https://foo.bar",
    "license": "private",
    "config": {
        "preferred-install": {
            "*": "dist"
        },
        "sort-packages": true,
        "bin-dir": "bin"
    },
    "extra": {
        "symfony": {
            "allow-contrib": false,
            "require": "4.4.*"
        }
    },
    "minimum-stability": "stable",
    "prefer-stable": true,
    "require": {
        "php": "^7.2"
    },
    "require-dev": {
    },
    "autoload": {
    },
    "replace": {
        "symfony/polyfill-ctype": "*",
        "symfony/polyfill-iconv": "*",
        "symfony/polyfill-php71": "*",
        "symfony/polyfill-php70": "*",
        "symfony/polyfill-php56": "*"
    },
    "scripts": {
        "auto-scripts": {
            "cache:clear": "symfony-cmd",
            "assets:install": "symfony-cmd",
            "security-checker security:check": "script",
            "assets:install %PUBLIC_DIR%": "symfony-cmd"
        },
        "check": [
            "@cs",
            "@test"
        ],
    },
    "conflict": {
        "symfony/symfony": "*"
    }
}
```

**Comment by Orkin** [comment]
@dunglas I just tried on a new folder to install this package on a new project. I mean, create a new folder, past the composer.json and `composer require --dev api-platform/schema-generator`. You will have the same error

**Comment by netprogs1** [comment]
I have similar errors. I'm not able to install api-platform/schema-generator package on Symfony 5.
On empty folder I run `symfony new project --full`, then `composer require --dev api-platform/schema-generator` with no success. Specifing a version, for example composer require --dev api-platform/schema-generator "2.2.1" does not work either.

**Comment by Orkin** [comment]
Any news about this issue ?

**Comment by roverwolf** [comment]
The lines
```
    "replace": {
        "symfony/polyfill-php70": "*",
        "symfony/polyfill-php71": "*"
    },
```
should be removed from `composer.json` as they will conflict with the same lines in the main project (and the schema-generator project does not actually replace those packages).

One can work around the issue by removing those lines from their project's `composer.json` file until a new version of `api-platform/schema-generator` fixes it.

**Comment by soyuka** [comment]
Can someone propose a patch for this?

**Comment by soyuka** [comment]
fixed in #189
